Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -592,6 +611,13 @@ theorem isOpenPosMeasure_of_mulLeftInvariant_of_regular [Regular μ] (h₀ : μ
 #align measure_theory.is_open_pos_measure_of_mul_left_invariant_of_regular MeasureTheory.isOpenPosMeasure_of_mulLeftInvariant_of_regular
 #align measure_theory.is_open_pos_measure_of_add_left_invariant_of_regular MeasureTheory.isOpenPosMeasure_of_addLeftInvariant_of_regular
 
+/-- A nonzero left-invariant inner regular measure gives positive mass to any open set. -/
+@[to_additive "A nonzero left-invariant inner regular measure gives positive mass to any open set."]
+theorem isOpenPosMeasure_of_mulLeftInvariant_of_innerRegular [InnerRegular μ] (h₀ : μ ≠ 0) :
```

Comment:
If you use `NeZero μ` instead of `μ ≠ 0` you can turn this theorem into an instance.